gluster-exporter/main.go

```diff
@@ -112,6 +69,10 @@ func main() {
 		return
 	}
 
+	// Set the Gluster Configurations used in glusterutils
+	glusterConfig.GlusterMgmt = *glusterMgmt
```

Comment:
Some of this config definition work overlaps with https://github.com/gluster/gluster-prometheus/pull/24
Anyway will re-base if this goes in first.

Reply:
This is now taken from CLI flags. Once Config PR merges then we can take from that.

Is it not possible to retain CLI flags with config PR? 1) Apply config from config file, 2) Overwrite config if CLI flag is passed

Reply:
We can think of this overwrite scenario I feel. So if CLI passes options it would overwrite the ones in config file. The only issue is I am exeacting `--config` and `--collectors-config` as CLI argument. With these should user really pass other command line arguments.
Or there is another option where we always expect global configuration and collectors configurations to be present in some path say `/etc/gluster-exporter/global.conf` and `/etc/gluster-exporter/collectors.conf` which get deplyed by rpm install. Now if user passes values of say `--gluster-mgmt` from CLI it overwrites the value provided in `global.conf`. Does this look a good suggestion @aravindavk ?

Reply:
Looks good. If we ship default configs then we need to change `--config` and `--collector-config` as optional arguments and also accept CLI args and overwrite config